vmdb/app/controllers/application_controller/current_user.rb

```diff
@@ -21,20 +21,27 @@ def current_user=(db_user)
       session[:userid]    = nil
       session[:username]  = nil
     end
+    self.current_group  = db_user.try(:current_group)
+    self.current_eligible_groups = db_user.try(:eligible_groups)
```

Comment:
Ugh. Great catch. This is wrong. It should be `db_user.miq_groups`.

Awh man. This was able to pass tests because the new rails put a `responds_to?()` into `try()`. Not a fan as it will let a bunch of errors slip by.

I guess since it is only displayed on `_user_options.html.haml`, this would have snuck under the radar for a while. :cold_sweat:

@dclarizio I'd like to remove `session[:eligible_groups]` and just use `current_user.miq_groups` in `_user_options.html.haml`. Thoughts?